tests/test_cfn_lint.py

```diff
@@ -2,11 +2,26 @@
 import unittest
 from pathlib import Path
 
+import mock
```

Comment:
none of this mocking is needed, just have to add `create_clients=False` to the test's `Config` instantiation

Reply:
Right. this is now outdated. thanks.

Reply:
don't see that this has been addressed

Reply:
This is actually necessary.